renderdoc/driver/gl/gl_debug.cpp

```diff
@@ -3762,7 +3763,9 @@ void GLReplay::InitPostVSBuffers(uint32_t eventID)
   GLuint vsoutBuffer = 0;
   gl.glGenBuffers(1, &vsoutBuffer);
   gl.glBindBuffer(eGL_ARRAY_BUFFER, vsoutBuffer);
-  gl.glNamedBufferDataEXT(vsoutBuffer, stride * primsWritten, data, eGL_STATIC_DRAW);
+  GLsizeiptr size = stride * primsWritten;
+  gl.glBufferData(eGL_ARRAY_BUFFER, size, data, eGL_STATIC_DRAW);
+  gl.m_Buffers[gl.GetResourceManager()->GetID(BufferRes(gl.GetCtx(), vsoutBuffer))].size = size;
```

Comment:
This one I don't understand, the emulation should kick in for `glNamedBufferDataEXT` to call into `glBufferData` the same way that it would on desktops that don't support `EXT_direct_state_access`.

The reason for requiring DSA functions for replay functions is just to reduce the GL API surface that could be called by replay code, and simplify mental load for coders. In particular book-keeping like handling `m_Buffers` then happens in as few places as possible.